sklearn/base.py

```diff
@@ -436,15 +436,15 @@ def fit_predict(self, X, y=None):
 
         Parameters
         ----------
-        X : ndarray, shape (n_samples, n_features)
+        X : ndarray of shape (n_samples, n_features)
```

Comment:
for the inputs here and below, this shouhld be array-like instead of ndarray

Reply:
Corrected.